tool/gravity/cli/status.go

```diff
@@ -170,6 +170,65 @@ func statusOnce(ctx context.Context, operator ops.Operator, operationID string,
 	return status, nil
 }
 
+// statusHistory collects cluster status history and prints out the information
+// to stdout.
+func statusHistory() error {
+	ctx, cancel := context.WithTimeout(context.Background(), defaults.StatusCollectionTimeout)
+	defer cancel()
+	timeline, err := statusapi.Timeline(ctx)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	for _, event := range timeline.GetEvents() {
+		printEvent(os.Stdout, event)
+	}
+
+	return nil
+}
+
+func printEvent(out io.Writer, event *pb.TimelineEvent) {
+	// stamp defines default timestamp format.
+	const stamp = "Jan _2 15:04:05 UTC"
+
+	w := new(tabwriter.Writer)
+	w.Init(out, 0, 8, 1, '\t', 0)
+
+	timestamp := event.GetTimestamp().ToTime()
+
+	switch event.GetData().(type) {
+	case *pb.TimelineEvent_ClusterDegraded:
+		eventString := fmt.Sprintf("[%s] Cluster Degraded\n", timestamp)
+		fmt.Fprintf(w, color.RedString(eventString))
```

Comment:
`fmt.Fprintf` will be flagged by lint as superfluous (should use `fmt.Fprint` instead).
Also, `color.XYZ` support formatting so explicit `fmt.Sprintf` is also redundant:
```go
fmt.Fprint(w, color.RedString("[%s] Cluster Degraded\n", timestamp))
```
here and below.